src/lib/services/CommandLineSupport.ts

```diff
@@ -74,7 +67,32 @@ export abstract class CommandLineSupport extends AsyncCreatable {
 
 		return new Promise<string>((res, rej) => {
 			const cp = cspawn.spawn(command, args);
-			this.monitorChildProcess(cp, res, rej);
+
```

Comment:
`monitorChildProcess` was moved into here, because it ends up being cleaner that way.